### ISSUE TITLE
NPC - Creature_template: Armor / Resistance

### ISSUE THREAD

**Issue by Kerafyrm** [scotty0100/wowascension-issues#299]
Name: Kerafyrm
Account Name: Kerafyrm
Map & Area: All Areas
How to replicate the bug: Kill any boss or mob in a dungeon instance.  Doing 45 minute Baron runs should not be this easy.

Extra Information:

Every boss and mob seems to go down so easily, regardless of the instance.  A 45 min Baron Stratholme raid was actually pretty difficult even with decent gear.  How 5 green geared people can somehow make it through quickly is something to be cautious of. 

Even bosses in BRD, LBRS, UBRS, and DM are a bit too easy.  Maybe an increase in their Spell Resist?  Armor?  Perhaps a +10% Increase in difficulty (in some way) will make it feel more true to vanilla. 

Just my two cents.

Kerafyrm <3

**Comment by Tobschinski** [comment]
You are correct, some mobs are missing their armor and resistances (although resistances are rare in WoW) , Baron Rivendare has 0 armor in particular. There are even mobs that do less damage than expected.

I can assure you, that these values are being looked into right now. However, there are more than 9000 different creatures, so it will take some time. Be prepared for a boost.

**Comment by Kerafyrm** [comment]
Take your time.  It's very sensitive on how to fine tune these Mobs, Bosses, and raid settings.  If you need test fodder, I'll help.

Kerafyrm <3
